source/YTEPs/YTEP-0033.rst

```diff
@@ -38,9 +39,11 @@ For this reason, many projects in the scientific python ecosystem have either al
 Proposed Solution
 ^^^^^^^^^^^^^^^^^
 
-I propose that we drop support for Python 2 in the major release of yt that happens after the next major release, yt 3.5. This release will be called yt 4.0, and may include a number of other major changes to the yt user experience, including `the demeshening <http://ytep.readthedocs.io/en/latest/YTEPs/YTEP-0032.html>`_ and `non-spatial indexing <http://ytep.readthedocs.io/en/latest/YTEPs/YTEP-0027.html>`_. In principle we could do a release and call it yt 3.6 that drops Python 2.7 support, however given that this is a major backwards incompatible change for the subset of our users who use Python 2.7, I think it would be best to signal this change with a major version number bump. We could also release yt 4.0 with the only breaking change being dropping of Python 2.7 support and then subsequently make other major changes that we are planning, however I feat that this approach would tax our limited development resources and draw out the transition further.
+I propose that we drop support for Python 2 in the major release of yt that happens after the next major release, yt 3.5. This release will be either yt 3.6 or yt 4.0, and may include a number of other major changes to the yt user experience, including `the demeshening <http://ytep.readthedocs.io/en/latest/YTEPs/YTEP-0032.html>`_ and `non-spatial indexing <http://ytep.readthedocs.io/en/latest/YTEPs/YTEP-0027.html>`_.
 
-Given our recent pace of development, I would expect yt 3.5 to be released summer 2018 and yt 4.0 will be released summer 2019. Some projects are treating their last release that supports Python 2.7 as a "long term support" (LTS) release. Given our development resources, I do not think that we have the bandwidth available to simultaneously support both yt 4.0 and yt 3.5 at the same level, however I do expect that we will make regular bugfix releases of the yt 3.5 series while yt 4.0 is under active development. This means that we will support yt 3.5 for approximately a year and officially end all support for Python 2.7 with the release of yt 4.0 in 2019.
+In the past, we had suggested that yt 3.6 would not exist, or would not drop python 2.7 support, but the delay in yt 4.0 release has changed this timeline.
+
+Some projects are treating their last release that supports Python 2.7 as a "long term support" (LTS) release. Given our development resources, I do not think that we have the bandwidth available to simultaneously support both yt 4.0 and yt 3.5 at the same level, however I do expect that we will make regular bugfix releases of the yt 3.5 series while yt 4.0 is under active development. This means that we will support yt 3.5 for approximately a year and officially end all support for Python 2.7 with the release of yt 4.0 in 2019.
 
```

Comment:
This is still talking about releasing yt-4.0 in 2019 and maintaining a 3.5 branch. Is that right? I was understanding you saying that we wouldn't maintain a python2 LTS branch because of our limited resources.

Reply:
Obviously we will not be releasing yt 4.0 in 2019, Nathan.  I left the majority of your text here as it was largely transcribed from your email message.

I can update the entirety of the YTEP if that is necessary.

Reply:
Can you? Right now it's not clear what the up-to-date info is.

Sorry if that came across as rude, I see now that you didn't actually edit this text and just adjusted the line lengths.

Reply:
I'm rewritin' it now and will update!